packages/test/test-end-to-end-tests/src/test/gc/gcUnreferencedTimestamp.spec.ts

```diff
@@ -659,14 +641,148 @@ describeCompat("GC unreferenced timestamp", "FullCompat", (getTestObjectProvider
 				);
 			});
 
+			/*
+			 * Similar to 6 but using a DDS attach op instead of a DataStore attach op
+			 *
+			 * Validates that we can detect references that were added via new DDSes before they are referenced
+			 * themselves, and then the reference from the new data store is removed.
+			 * 1. Summary 1 at t1. V = [A*, C]. E = []. C has unreferenced time t1.
+			 * 2. Data store B is created and referenced from A. E = [A.root -> B].
+			 * 3. DDS B.1 under B is created but not referenced. E = [A.root -> B].
+			 * 4. Add reference from DDS B.1 to C. E = [A -> B].
+			 * 5. Op adds reference from B's root to DDS B.1. E = [A.root -> B, B.root -> B.1, B.1 -> C].
+			 * 6. Op removes reference from DDS B.1 to C. E = [A.root -> B, B.root -> B.1].
+			 * 7. Summary 2 at t2. V = [A*, B, C]. E = [A.root -> B, B.root -> B.1]. C has unreferenced time t2.
+			 *
+			 * Validates that the unreferenced time for C is t2 which is > t1.
+			 */
+			it(`Scenario 8 - Node re-referenced via new unreferenced DDS attaching`, async () => {
```

Comment:
Another option would be to move these scenarios under a different `describeCompat` or boundary so we don't lose any of our original coverage.

Reply:
I spent a couple min thinking about this, it's a bit annoying to do since there's so much shared helpers / globals within that top-level describe.  Also it's an awkward time since specifying min version is broken. So I'm going to let it go for now, knowing that we'll get min version coverage for rc.1.0 onward when that's fixed.